doc/content/the-things-stack/monitor/receivers-and-profiles/_index.md

```diff
@@ -0,0 +1,34 @@
+---
+title: "Receiving alert notifications"
```

Comment:
I think this section should be "Configure how you receive alerts from The Things Stack".

The intro sentence can then say that we use alert notification receivers and profiles to configure this and then the docs can continue.

Reply:
I think the name is too verbose for a subsection of a subsection of a subsection, but I have restructured this a bit.